### DISCUSSION TITLE
ESP32-CAM with crazyflie

### DISCUSSION THREAD

**Discussion by cuber-1** [bitcraze/discussions#1461] [General]
Rather than using an AI deck for a camera, I'm thinking of using an ESP32-CAM with a microUSB port attached to run an AI model and process video in conjunction with the Crazyflie drone. Is this even possible? How would I send information to the drone? Could I do this on a Windows device? How do I power the camera with the crazyflie drone? Will the camera be compatible with the Crazyflie software?

**Comment by gemenerik** [comment]
Integrating an ESP32-CAM with a Crazyflie drone is highly complex and unsupported by us. You'd need to figure out how to connect and power it correctly, establish a unified communication protocol (such as implementing CPX on the ESP32-CAM), and ensure compatibility with Crazyflie software. This involves substantial hardware modifications, firmware development, and software integration. One way to get started is to compare it with the schematics of our boards, but note that we do not offer support for this configuration, so it would require a lot of independent troubleshooting and development.
